pkg/loki/loki.go

```diff
@@ -259,6 +264,18 @@ func (c *Config) Validate() error {
 	return nil
 }
 
+func ValidateConfigTimeouts(c *Config) error {
```

Comment:
This should probably be moved to a different package. Maybe the validation package.